apps/web/src/pages/studio-onboarding/components/Background.tsx

```diff
@@ -0,0 +1,27 @@
+import { css } from '@novu/novui/css';
+export const Background = () => (
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
💭 **thought (non-blocking):** Perhaps we can use the [CSS solution](https://github.com/novuhq/novu/blob/0727c3c77df163382330a505be11e3749a5e124a/apps/web/src/studio/components/workflows/WorkflowsDetailPage.tsx#L44) I made here? We can run it by Nikolay since it's slightly (negligibly unless you're looking for it) different, but my thought is that it's more easily reusable, composable, etc‏